### DISCUSSION TITLE
i got a problem with my account

### DISCUSSION THREAD

**Discussion by JoaquiinAguilar** [community/community#69338] [GitHub Education]
### Hi everyone,

It says that my name should be the same as my academic affiliation document, and my name is the same idk why keeps doing the same problem
![Captura de pantalla (92)](https://github.com/community/community/assets/92810537/d67ef323-e66b-4e40-a37d-79266c588170)

**Comment by mantasu** [comment]
It could be that there are some non-English characters or a translation might be needed - see the answer [here](https://github.com/orgs/community/discussions/68725#discussioncomment-7163466).

If you haven't yet been approved and the issue persists, you can always open a [ticket](https://support.github.com/request/education).

**Reply by mantasu** [reply]
If you haven't received the benefits yet, I saw that staff is suggesting reapplying in some cases, for example [here](https://github.com/orgs/community/discussions/69133#discussioncomment-7185814), so worst case I would suggest to just re-raise the ticket and reapply :)
